src/lib/Sympa/Message.pm

```diff
@@ -63,6 +63,8 @@ use Sympa::Tools::Password;
 use Sympa::Tools::SMIME;
 use Sympa::Tools::Text;
 use Sympa::User;
+use CGI;
+use URI::Find::Schemeless;
```

Comment:
URI::Find::Schemeless should also be added to the `cpanfile` file on the top of source tree as a `requires` section.